conf/kops/helmfile.yaml

```diff
@@ -0,0 +1,542 @@
+#
+# Master Helmfile 
+#
+#      by Cloud Posse <hello@cloudposse.com>
+#
+
+
+#
+# Usage:
+#   
+#   helmfile sync --selector namespace=kube-system,name!=chart-repo
+#
+# References: 
+#   - https://github.com/roboll/helmfile
+#
+
+repositories:
+# Stable repo of official helm charts
+- name: "stable"
+  url: "https://kubernetes-charts.storage.googleapis.com"
+
+# Cloud Posse incubator repo of helm charts
+- name: "cloudposse-incubator"
+  url: "https://charts.cloudposse.com/incubator/"
+
+# CoreOS Stable helm charts
+- name: "coreos-stable"
+  url: "https://s3-us-west-2.amazonaws.com/coreos-charts/stable"
+
+releases:
+
+################################################################################
+## Chart Museum ################################################################
+################################################################################
+
+- name: chartmuseum
+  namespace: kube-system
+  labels:
+    name: chartmuseum
+    namespace: kube-system
+    component: platform
+    required: false
+  chart: stable/chartmuseum
+  version: 1.3.1
+  set: 
+    - name: "replicaCount"
+      value: "1"
+
+    # disable all routes prefixed with /api
+    - name: "env.open.DISABLE_API"
+      value: "false"
+
+    # allow chart versions to be re-uploaded
+    - name: "env.open.ALLOW_OVERWRITE"
+      value: "true"
+
+    ## Required: CHARTMUSEUM_BASIC_AUTH_USER; e.g. server
+    - name: "env.secret.BASIC_AUTH_USER"
+      value: '{{ env "CHARTMUSEUM_BASIC_AUTH_USER" }}'
+
+    # password for basic http authentication
+    ## Required: CHARTMUSEUM_BASIC_AUTH_PASS
+    - name: "env.secret.BASIC_AUTH_PASS"; e.g. 15041434-3DC2-42F8-9358-F8AED8780A66
+      value: '{{ env "CHARTMUSEUM_BASIC_AUTH_PASS" }}'
+
+    ### Resources
+    - name: "resources.limits.cpu"
+      value: "100m"
+
+    - name: "resources.limits.memory"
+      value: "512Mi"
+
+    - name: "requests.cpu" 
+      value: "5m"
+
+    - name: "requests.memory"
+      value: "256Mi"
+
+    ### Persistence
+    - name: "persistence.enabled"
+      value: "true"
+
+    - name: "persistence.accessMode"
+      value: "ReadWriteOnce"
+
+    - name: "persistence.size"
+      value: "1Gi"
```

Comment:
is 1G enough? Many chart versions will be built during unlimited staging env deployments, including every tag